### PR TITLE
Completed React accordion component - Daniel Rosenbaum.

### PR DESCRIPTION
**Pull request by danielwrosenbaum**
Accordion

### CODE REVIEW

File: react-accordion-component/src/accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    content: 'Extra Content'
+  }
+];
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = {
+      isClicked: false,
```
Comment (thebearingedge) [inline react-accordion-component/src/accordion.jsx:26]: This `isClicked` isn't really used for anything.

File: react-accordion-component/src/accordion.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  },
+  {
+    title: 'JavaScript',
+    content: 'JavaScript, often abbreviated as JS, is a high-level, interpreted programming language that conforms to the ECMAScript specification. JavaScript has curly-bracket syntax, dynamic typing, prototype-based object-orientation, and first-class functions.'
+  },
+  {
+    title: 'Extra Heading',
+    content: 'Extra Content'
+  }
+];
```
Comment (thebearingedge) [inline react-accordion-component/src/accordion.jsx:19]: This array should be passed as a prop in your `index.jsx`.

File: react-accordion-component/src/accordion.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    const isClicked = this.state.isClicked;
+    if (!isClicked) {
+      this.setState({
+        isClicked: true,
+        target: event.target.id
+      });
+    } else {
+      this.setState({
+        isClicked: false,
+        target: event.target
```
Comment (thebearingedge) [inline react-accordion-component/src/accordion.jsx:41]: Try `null` instead of `event.target`. In general, we do not store DOM elements in state.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great, just a little bit of clean up needed.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
That `isClicked` can go.

**Comment by danielwrosenbaum** [comment]
> That `isClicked` can go.

right sorry! changed that but then had to go back to fix something else and forgot to switch that back. will fix now!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**